docs/embed/commons-docs-swagger.html

```diff
@@ -0,0 +1,12 @@
+<div id="commons-swagger-ui"></div>
+<script>
+    $(document).ready(function () {
+        const options = {
+            version: '6.4.x-devx-SNAPSHOT',
```

Comment:
```suggestion
            version: '6.4.x-devx-SNAPSHOT',
```
Could you let us know after removing this FB? This part of the documentation won't be broken? You have to keep in mind that we don't keep product snapshots forever nor FB snapshots